### ISSUE TITLE
Will this work with AngularJS?

### ISSUE THREAD

**Issue by jorgedavila25** [alexeagle/angular-bazel-example#103]
Hi,

I'm currently working on a project where I have to build an AngularJS project with Bazel. Doesn't seem like it, but I'm curious to know if ABC is compatible w/ AngularJS 1.6. 

Thank you in advance

**Comment by alexeagle** [comment]
Yes, though we don't have an example yet. For AngularJS there is no ahead-of-time compiler, so if you author your app in TypeScript you can use the ts_library rule to compile it.

**Comment by alexeagle** [comment]
Moved to https://github.com/bazelbuild/rules_nodejs/issues/1129
